cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/TimeScheduler.java

```diff
@@ -240,8 +267,7 @@ public void suspendSchedule(Id.Program program, SchedulableProgramType programTy
     throws NotFoundException, SchedulerException {
     checkInitialized();
     try {
-      scheduler.pauseTrigger(new TriggerKey(AbstractSchedulerService.scheduleIdFor(program, programType,
-                                                                                   scheduleName)));
+      scheduler.pauseTrigger(getGroupedTriggerKey(program, programType, scheduleName));
```

Comment:
what happens if someone attempts to pause a schedule that is still in the new paused triggers group? Will he get an error: schedule not found? Or will he get: schedule is already paused?